threat_intel/util/http.py

```diff
@@ -114,7 +153,7 @@ class MultiRequest(object):
     _VERB_GET = 'GET'
     _VERB_POST = 'POST'
 
-    def __init__(self, default_headers=None, max_requests=20, rate_limit=0, req_timeout=None, max_retry=10):
+    def __init__(self, default_headers=None, max_requests=20, rate_limit=0, req_timeout=None, max_retry=10, total_retry=100):
```

Comment:
It maybe worth a while to break this line as well.
We should actually add a code style check to the tox file to not allow lines longer than 80 characters.